networkx/algorithms/approximation/steinertree.py

```diff
@@ -0,0 +1,76 @@
+import networkx as nx
+from itertools import combinations, chain
```

Comment:
According to [PEP8][1], standard library imports should be first, then library-specific imports after, separated by a blank line.

[1]: https://www.python.org/dev/peps/pep-0008/#imports